src/pudl/output/ferc1.py

```diff
@@ -2043,27 +2051,27 @@ def in_explosion_tables(table_name: str, in_explosion_table_names: list[str]) ->
 class XbrlCalculationForestFerc1(BaseModel):
     """A class for manipulating groups of hierarchically nested XBRL calculations.
 
-    We expect that the facts reported in high-level FERC tables like
+    We expect that the facts reported in less granular FERC tables like
     :ref:`core_ferc1__yearly_income_statements_sched114` and
     :ref:`core_ferc1__yearly_balance_sheet_assets_sched110` should be
     calculable from many individually reported granular values, based on the
     calculations encoded in the XBRL Metadata, and that these relationships should have
-    a hierarchical tree structure. Several individual values from the higher level
+    a hierarchical tree structure. Several individual values from the less granular
     tables will appear as root nodes at the top of each hierarchy, and the leaves in
     the underlying tree structure are the individually reported non-calculated values
-    that make them up. Because the top-level tables have several distinct values in
-    them, composed of disjunct sets of reported values, we have a forest (a group of
-    several trees) rather than a single tree.
+    that make them up (i.e. the most granular values). Because the less granular tables
+    have several distinct values in them, composed of disjunct sets of reported values,
+    we have a forest (a group of several trees) rather than a single tree.
```

Comment:
This phrasing is still a little confusing. Are you saying that values from composite "less granular" tables can be both roots and leaves?

The word "underlying" still sounds too "rooty" for a "leaf" could take it out entirely and say something like: "Composite values (root nodes) from less granular tables form the base of the hierarchy from which the most granular calculation components (leaves) arise. Because the less granular tables have several distinct values in them and are composed of a disjunct sets of reported values, our result is a forest (a group of several trees) rather than a single tree."

Reply:
agreed that underlying is not a great word here. I don't think your suggestion here is totally correct. A ton of the nodes on the tree are composite values - everything except the leaves are composite notes. and as i mentioned in the other thread i don't think "base of hierarchy" is accurate.

also, a deep re-write of this doc is something we just said on the phone was oos for this pr. my edits rn on this are simply to remove "underlying"

Reply:
could say "form the base of the tree" instead of the hierarchy. I see why it's weird to say "base of the hierarchy"

Reply:
I don't think this necessarily negates the fact that non-leaf values can be calculated values. But you're right, there's no need to get stuck here. You can keep it as is if you're more comfortable with that!!

Reply:
I think we did discuss the importance of having a sentence that conveys the relationship between seed/root/leaves/tree/forest relationship (in addition to linking to the networkx docs). Def don't want to re-write all the tree language, just clarify it once.

Reply:
okay im going to remove the hierarchy bit altogether. i don't think it adds anything and any other way besides "root" and "less granular" is complicating.